tutorials/frontend/deploy_ssd_gluoncv.py

```diff
@@ -45,8 +45,15 @@
 #   follow the :ref:`tune_relay_x86` to tune x86 CPU and
 #   :ref:`tune_relay_arm` for arm CPU.
 #
-#   To get best performance fo SSD on Intel graphics,
-#   change target argument to 'opencl -device=intel_graphics'
+#   To get best inference performance on Intel graphics,
+#   change target argument to :code:`opencl -device=intel_graphics`,
+#   except for Mac, target needs to be set to `opencl` only.
```

Comment:
not sure whether it is clear for native speaker ... "except for Mac, target needs to be set to opencl only."  means set "opencl"  for mac, or for non-mac?
shall we also mention intel extension on mac?

Reply:
Updated, lemme know how it looks now.